src/components/DpDraggable/DpDraggable.stories.mdx

```diff
@@ -0,0 +1,29 @@
+import { Meta } from '@storybook/addon-docs'
+import DpDraggable from './DpDraggable'
+
+<Meta
+    title="Components/Draggable"
+    component={DpDraggable}
+/>
+
+# Draggable
+
+The Draggable component can change the order of items within a list.
+
+
+## Default usage
+
+```html
+<dp-draggable
```

Comment:
I think that `v-model` belongs here, too.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/497/commits/089a7a49c5d37e64eb42122ba6901f6eb3cc4769